lib/cocoapods-binary-cache/pod-rome/xcodebuild_command.rb

```diff
@@ -173,6 +198,14 @@ def framework_path_of(target, sdk)
       "#{target_products_dir_of(target, sdk)}/#{target.product_module_name}.framework"
     end
 
+    def dsyms_path_of(target, sdk)
+      "#{target_products_dir_of(target, sdk)}/*.dSYM"
```

Comment:
How about returning an array here:
```rb
Dir["#{target_products_dir_of(target, sdk)}/*.dSYM"]
```
Same for the code inside `bcsymbolmaps_path_of`